src/guiguts/tools/pptxt.py

```diff
@@ -1373,12 +1373,12 @@ def report_multiple_occurrences_on_line(
     error_start = matches[0].start()
     error_end = matches[-1].end()
     # Add record to the dialog with prefix if repeated matches.
+    repeat_str = f"(x{len(matches)}) " if len(matches) > 1 else ""
```

Comment:
I was surprised this file is involved in this PR. Is this part of the ebookmaker change?

Reply:
Yes - it's a consequence of making severities more flexible. Previously the repeat string hijacked the error_prefix mechanism, but that was a bad/lazy decision on my part. So this edit is undoing that laziness, now that severity and error prefix can be set independently.